spec/index.html

```diff
@@ -631,6 +652,247 @@ <h2>Streams</h2>
+
+      <p>A YAML <a>block sequence</a>:</p>
+      <pre data-transform="updateExample"
+           data-content-type="application/yaml">
+        <!--
+          - one
+          - two
+        -->
+      </pre>
+
+      <p>A YAML <a>flow sequence</a>:</p>
+      <pre data-transform="updateExample"
+           data-content-type="application/yaml">
+        <!--
+          [one, two]
+        -->
+      </pre>
+    </aside>
+
+    <p>The conversion process represented here is compatible with
+      the description of <a data-cite="YAML#constructing-native-data-structures">Constructing Native Data Structures</a> in [[YAML]].</p>
+
+    <section id="convert-stream" class="algorithm">
+      <h3>Converting a <a>YAML stream</a></h3>
+
+      <p>A <a>YAML stream</a> is composed of zero or more <a>YAML documents</a>.</p>
+
+      <p class="issue" data-number="63">
+        <a>YAML streams</a> may correspond more directly to
+        [[[RFC7464]]], which are not presently part of the
+        JSON-LD representation model.
+        The description here more closely aligns with how JSON-LD
+        interprets <a data-cite="JSON-LD11-API#html-content-algorithms">HTML Scripts</a>.
+      </p>
+      <ol>
+        <li>Set <var>stream content</var> to an empty <a>array</a>.</li>
+        <li>If the <a>stream</a> is empty,
+          set <var>stream content</var> to an empty <a>array</a>.</li>
+        <li>Otherwise, if the <a>stream</a> contains a single <a>YAML document</a>,
+          set <var>stream content</var> the result of
+          <a href="#convert-document" class="sectionRef"></a>.</li>
+        <li>Otherwise: for each <var>document</var> in the <a>stream</a>:
+          <ol>
+            <li>Set <var>doc</var> to the result of <a href="#convert-document" class="sectionRef"></a> for <var>document</var>.</li>
+            <li>If <var>doc</var> is an <a>array</a>,
+              merge it to the end of <var>stream content</var>.</li>
+            <li>Otherwise, append <var>doc</var> to <var>stream content</var></li>
+          </ol>
+        </li>
+        <li>The conversion result is <var>stream content</var>.</li>
+      </ol>
+
+      <p>Any error reported in a recursive processing step results
+        in the failure of this processing step.</p>
+    </section>
+
+    <section id="convert-document">
+      <h3>Converting a <a>YAML document</a></h3>
+
+      <p>From the <a data-cite="YAML#">YAML grammar</a>,
+        a YAML document MAY be preceded by a
+        <a data-cite="YAML#rule-l-document-prefix">Document Prefix</a>
+        and/or a set of
+        <a data-cite="YAML#rule-l-directive">directives</a>
+        followed by a
+        <a data-cite="YAML#rule-l-bare-document">YAML bare document</a>,
+        which is composed of a single <a>node</a>.
+      </p>
+
+      <ol>
+        <li>Create an empty may <var>named nodes</var>,
+          used the associate an anchor name with the <a>node</a>
+          on which it is defined.</li>
```

Comment:
Guessing wildly at what was intended to be said here...
```suggestion
        <li>Create an empty <var>named node</var>,
          and associate an anchor name with the <a>node</a>
          on which it is defined.</li>
```

Reply:
I sure screwed up that sentence! 

At this point, we don't have a node to create such an association, only a `map` used for saving them in deeper steps.

It should have been the following:

```suggestion
        <li>Create an empty <a>map</a> <var>named nodes</var>,
          used the associate an anchor name with the <a>node</a>
          on which it is defined.</li>
```

Reply:
@gkellogg — That helps with the first line, but the two that follow still need some changes to make the whole sentence coherent.